service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ComplyWithDirectionsControllerTest.java

```diff
@@ -70,37 +71,76 @@ void aboutToStartCallbackShouldAddAllPartiesDirectionsIntoSeparateRoleCollection
 
     @SuppressWarnings("unchecked")
     @Test
-    void aboutToSubmitCallbackShouldAddResponseToResponses() throws Exception {
+    void aboutToSubmitShouldAddResponseToStandardDirectionOrderWhenEmptyServedCaseManagementOrders() throws Exception {
```

Comment:
I'm confused, why is this test mentioning doing something to SDO based on CMO, aren't they seperate

Reply:
Seems like this test is only checking things with SDO

Reply:
Ahh, I see now the service tests based on the list being empty, maybe rename the method to be `aboutToSubmitShouldAddResponseToStandardDirectionOrderWhenThereAreNoServedCaseManagementOrders`